src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1491,6 +1491,117 @@ def cleanse_crash(self,
       return result
 
 
+class AndroidEmulatorLibFuzzerRunner(AndroidLibFuzzerRunner):
+  """Android emulator libFuzzer runner."""
+
+  # pylint: disable=super-init-not-called
+  def __init__(self, executable_path, build_directory, default_args=None):
+    """Inits the AndroidEmulatorLibFuzzerRunner.
+
+    Args:
+      executable_path: Path to the fuzzer executable.
+      build_directory: A MinijailChroot.
+      default_args: Default arguments to always pass to the fuzzer.
+    """
+    self.exec_path = executable_path
```

Comment:
nit: rather than naming these slightly differently, do

```python
self._init_args = {
   'executable_path': executable_path,
  ...
}
```

Then later you can do

```python
super().__init__(**self._init_args)
```

Reply:
I reworked this to call the parent `__init__()` function here, but I do still need the `build_dir` to be saved to start the emulator.